app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/response/IncludedResponse.kt

```diff
@@ -48,11 +49,20 @@ fun IncludedQuestionResponse.toSurveyQuestion() =
         title = attributes?.text.orEmpty().trim(),
         displayOrder = attributes?.displayOrder ?: 0,
         shortText = attributes?.shortText.orEmpty(),
-        pick = attributes?.pick.orEmpty(),
+        pick = attributes?.pick.toSurveyQuestionPickType(),
         questionDisplayType = attributes?.displayType.orEmpty().getQuestionDisplayType(),
         answers = emptyList()
     )
 
+private const val SINGLE_PICK = "one"
+
+fun String?.toSurveyQuestionPickType(): SurveyQuestionPickType {
+    this?.let { pick ->
+        return if (pick == SINGLE_PICK) SurveyQuestionPickType.SINGLE else SurveyQuestionPickType.MULTIPLE
+    }
+    return SurveyQuestionPickType.NONE
+}
```

Comment:
We can be more explicit here and simplify the logic.
If we add another constant for `MULTIPLE_PICK`, we can then do something like this
```suggestion
fun String?.toSurveyQuestionPickType(): SurveyQuestionPickType {
    return when (this) {
        SINGLE_PICK -> SurveyQuestionPickType.SINGLE
        MULTIPLE_PICK -> SurveyQuestionPickType.MULTIPLE
        else -> SurveyQuestionPickType.NONE
    }
}
```
What do you think? 😄